src/lib/template-result.ts

```diff
@@ -20,6 +20,28 @@ import {reparentNodes} from './dom.js';
 import {TemplateProcessor} from './template-processor.js';
 import {boundAttributeSuffix, lastAttributeNameRegex, marker, nodeMarker} from './template.js';
 
+let policy: Pick<TrustedTypePolicy, 'createHTML'>|undefined;
+
+/**
+ * Turns the value to trusted HTML. If the application uses Trusted Types the
+ * value is transformed into TrustedHTML, which can be assigned to execution
+ * sink. If the application doesn't use Trusted Types, the return value is the
+ * same as the argument.
+ */
+function convertConstantTemplateStringToTrustedHTML(value: string): string|
+    TrustedHTML {
+  // tslint:disable-next-line
+  const w = window as any
+  // TrustedTypes have been renamed to trustedTypes
+  // (https://github.com/WICG/trusted-types/issues/177)
+  const trustedTypes =
+      (w.trustedTypes || w.trustedTypes) as TrustedTypePolicyFactory;
```

Comment:
I think one of these should have been `w.TrustedTypes`?

Reply:
Yeah, fixed in #1003